src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Making your website "cross-origin isolated" using COOP and COEP
+subhead: >
+  Use COOP and COEP to set up a cross-origin isolated environment and enable
+  powerful features like `SharedArrayBuffer`, `performance.measureMemory()` or
+  the JS Self-Profiling API.
+description: >
+  Some web APIs increase the risk of side-channel attacks like Spectre. To
+  mitigate that risk, browsers offer an opt-in-based isolated environment called
+  cross-origin isolated. Use COOP and COEP to set up such an environment and
+  enable powerful features like `SharedArrayBuffer`,
+  `performance.measureMemory()` or the JS Self-Profiling API.
+authors:
+  - agektmr
+hero: hero.png
+date: 2020-04-13
+updated: 2020-04-14
+tags:
+  - post
+  - security
+---
+Some web APIs increase the risk of side-channel attacks like Spectre. To 
+mitigate that risk, browsers offer an opt-in-based isolated environment called 
+cross-origin isolated. With a cross-origin isolated state, the webpage will be 
+able to use privileged features including:
+
+* [`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+  (required for WebAssembly Threads)
+* [`performance.measureMemory()`](/monitor-total-page-memory-usage/)
+* [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/)
+
+The cross-origin isolated state also prevents modifications of 
+`document.domain`. (Being able to alter `document.domain` allows 
+communication between same-sites and has been considered a loophole in the 
```

Comment:
I don't think "same-sites" is a thing. Replace with "same-site documents"?

Reply:
+1